### DISCUSSION TITLE
Can we support the way of diffrent testcase pass rate caculation for different scenarios of skip testcases

### DISCUSSION THREAD

**Discussion by zjd2626** [pytest-dev/pytest#8912] [Ideas]
[https://github.com/allure-framework/allure2/discussions/1348](url)
Maybe some scenarios is that some testcases mark as skip because some feature have issue or haven't been implemented,in this case, when counting test results, you may want to include skip testcases in all executed cases. Another scenario is that it does not need to be executed because conditions is not meet. In this case, when counting the test results, we hope to exclude the skip case from all executed cases. Is it possible to add a parameter to pytest.skip to support these two situations?

**Comment by RonnyPfannschmidt** [comment]
Im under the impression you want a new outcome similar to skip and xfail

A new unclear meaning should not be shoehorned into skip

**Reply by RonnyPfannschmidt** [reply]
The original discussion is no longer accessible
